memqueue/queue.go

```diff
@@ -127,6 +127,10 @@ func (q *Queue) Add(msg *msgqueue.Message) error {
 	return q.enqueueMessage(msg)
 }
 
+func (q *Queue) Len() (int, error) {
+	return int(q.Processor().Stats().Buffered), nil
```

Comment:
`return q.Processor().Len(), nil`. Processor should use `len(p.ch)` to get number of queued messages